src/app/cube/cube.component.html

```diff
@@ -11,4 +11,5 @@
     </div>
     <cube-footer></cube-footer>
   </div>
-</div>
\ No newline at end of file
+</div>
+<div id="learning-object-download"></div>
```

Comment:
This was placed here so that any component in the cube-component can trigger a download. 

It was also rewritten (from the original: `<div [id]="iframeParent"></div>`) since the id is in fact static, as it is only used as a container to inject iframes into.